src/connector/sql/sqlGeneration.ts

```diff
@@ -233,57 +317,115 @@ export const scalarComparisonOperatorMappings: ScalarOperatorMappings = {
   },
   String: {
     comparison: {
+      is_null: {
+        name: "IS_NULL",
+        isInfix: false,
+        isUnary: true,
+      },
+
       eq: {
         name: "=",
         isInfix: true,
+        isUnary: false,
       },
       neq: {
         name: "!=",
         isInfix: true,
+        isUnary: false,
       },
       gt: {
         name: ">",
         isInfix: true,
+        isUnary: false,
       },
       lt: {
         name: "<",
         isInfix: true,
+        isUnary: false,
       },
       gte: {
         name: ">=",
         isInfix: true,
+        isUnary: false,
       },
       lte: {
         name: "<=",
         isInfix: true,
+        isUnary: false,
       },
       contains: {
         name: "CONTAINS",
         isInfix: false,
+        isUnary: false,
       },
       endswith: {
         name: "ENDSWITH",
         isInfix: false,
+        isUnary: false,
       },
       regexmatch: {
         name: "REGEXMATCH",
         isInfix: false,
+        isUnary: false,
       },
       startswith: {
         name: "STARTSWITH",
         isInfix: false,
+        isUnary: false,
       },
     },
   },
 };
 
+export function getScalarType(column: Column): string {
+  if (column.nestedField) {
+    return getNestedScalarType(column.nestedField);
+  } else {
```

Comment:
We don't an `else {}` here (a little less nesting)

Reply:
Done.